src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1427,10 +1435,8 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
             {
                 _TrySendMouseEvent(point);
                 args.Handled(true);
-                return;
             }
-
-            if (_focused && point.Properties().IsLeftButtonPressed())
+            else if (_focused && point.Properties().IsLeftButtonPressed())

```

Comment:
this all seems unrelated. But if we're doing this, I think you can remove 1437 too? We set args.Handled to true on line 1519.